tests/samplers_tests/test_gp.py

```diff
@@ -82,22 +92,21 @@ def constraints_func(_: FrozenTrial) -> Sequence[float]:
         warnings.simplefilter("ignore", optuna.exceptions.ExperimentalWarning)
         sampler = GPSampler(n_startup_trials=2, constraints_func=constraints_func)
 
-    study = optuna.create_study(direction="minimize", sampler=sampler)
+    def objective(
+        trial: optuna.Trial | optuna.trial.FrozenTrial,
+    ) -> tuple[float] | tuple[float, float]:
+        x = trial.suggest_float("x", 0, 1)
+        if n_objectives == 1:
+            return (x,)
```

Comment:
Returning a `float` instead of `tuple[float]` is sufficient for the single-objective optimization.
```suggestion
    ) -> float | tuple[float, float]:
        x = trial.suggest_float("x", 0, 1)
        if n_objectives == 1:
            return x
```